editor/src/components/editor/editor-component.tsx

```diff
@@ -456,7 +456,7 @@ export function EditorComponent(props: EditorProps) {
   )
 }
 
-const EditorCursorComponent = React.memo(() => {
+export const EditorCursorComponent = React.memo(() => {
```

Comment:
This should probably be moved out of this file now and into the `canvas/controls` folder somewhere

Reply:
fixed